config/formats.ts

```diff
@@ -2756,21 +2765,32 @@ export const Formats: FormatList = [
 
 		mod: 'gen5',
 		team: 'random',
-		ruleset: ['Obtainable', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod'],
+		ruleset: ['Obtainable', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod', 'Illusion Level Mod', 'Zarel\'s Blessing'],
+		zarelsBlessing: {
+			'Unown': {'spa': 1},
+		},
 	},
 	{
 		name: "[Gen 4] Random Battle",
 
 		mod: 'gen4',
 		team: 'random',
-		ruleset: ['Obtainable', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod'],
+		ruleset: ['Obtainable', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod', 'Zarel\'s Blessing'],
+		zarelsBlessing: {
+			'Ditto': {'spe': 1},
+			'Unown': {'spa': 1},
+		},
 	},
 	{
 		name: "[Gen 3] Random Battle",
 
 		mod: 'gen3',
 		team: 'random',
-		ruleset: ['Standard'],
+		ruleset: ['Standard', 'Zarel\'s Blessing'],
+		zarelsBlessing: {
```

Comment:
i kinda don't like how this isn't customizable at all so people can't experiment with it in custom challenges. can we look into that eventually?

Reply:
We do have the complex ruleset system we use for I think EVs somewhere. It might be too complex though.

Reply:
I can try to whip something up and discuss it in #sim-dev

Reply:
Though that's probably something for a different PR.

Reply:
Absolutely we're open to that from a rands policy perspective